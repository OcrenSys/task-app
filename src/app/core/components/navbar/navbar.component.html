<mat-toolbar color="primary">
  <span>Task App</span>
</mat-toolbar>
