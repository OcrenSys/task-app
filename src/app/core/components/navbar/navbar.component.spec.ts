import { ComponentFixture, TestBed } from '@angular/core/testing';

import { NavbarComponent } from './navbar.component';

describe('NavbarComponent', () => {
  let component: NavbarComponent;
  let fixture: ComponentFixture<NavbarComponent>;

  beforeEach(() => {
    fixture = TestBed.createComponent(NavbarComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  test('should create', () => {
    expect(component).toBeTruthy();
  });

  test('should match snapshot', () => {
    const compiled = fixture.nativeElement as HTMLElement;
    expect(compiled).toMatchSnapshot();
  });
});
