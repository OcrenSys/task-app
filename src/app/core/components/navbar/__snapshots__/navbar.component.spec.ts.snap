// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavbarComponent should match snapshot 1`] = `
<div
  id="root1"
>
  <mat-toolbar
    class="mat-toolbar mat-primary mat-toolbar-single-row"
    color="primary"
  >
    <span>
      Task App
    </span>
  </mat-toolbar>
</div>
`;
